build-logic/binary-compatibility/src/test/groovy/gradlebuild/binarycompatibility/PublicAPIRulesTest.groovy

```diff
@@ -485,6 +485,7 @@ class PublicAPIRulesTest extends Specification {
             Map<String, ?> getUserData() {
                 [
                     currentVersion: '11.38',
+                    currentMasterVersion: '11.38',
```

Comment:
🤔 Should it be tested with currentMasterVersion != currentVersion?

Reply:
We can also change that yes

Reply:
Done